portality/tasks/anon_export.py

```diff
@@ -136,7 +136,8 @@ def run_anon_export(tmpStore, mainStore, container, clean=False, limit=None, bat
             transform = anonymisation_procedures[type_]
```

Comment:
There's a bunch of modifications here which may come from @Steven-Eardley 's work on the anon export, I'm not sure.  Are these two bits of work combined in this PR?

Reply:
I guess the change not related to `anon export` branch.

I did refactoring to extract and apply some method on this file anon_export.py.